modules/govuk/manifests/apps/efg_training.pp

```diff
@@ -28,7 +28,7 @@
 ) {
   validate_string($vhost_name)
 
-  $app_name = 'efg_training'
+  $app_name = 'efg-training'
```

Comment:
I made a similar backwards change here: https://github.com/alphagov/govuk-puppet/commit/02b24011f57a2161bc8a92075d50b0dd2ab1a015

I thought that was a fix for the `govuk::app::envvar` thing, so if so that'll break, otherwise I was just doing random things it seems